flow/project.py

```diff
@@ -1007,75 +1012,95 @@ def eligible(self, job, ignore_conditions=IgnoreConditions.NONE):
             A :class:`~signac.contrib.job.Job` from the signac workspace.
-        :type job:
-            :class:`~signac.contrib.job.Job`
-        :param ignore_conditions:
-            Specify if pre and/or post conditions are to be ignored while checking eligibility.
-            The default is :class:`IgnoreConditions.NONE`.
-        :type ignore_conditions:
-            :class:`~.IgnoreConditions`
-        :return:
+        ignore_conditions : :class:`~.IgnoreConditions`
+            Specify if preconditions and/or postconditions are to be ignored
+            while checking eligibility.  The default is
+            :class:`IgnoreConditions.NONE`.
+
+        Returns
+        -------
+        bool
             Whether the group is eligible.
-        :rtype:
-            bool
+
         """
         return self._eligible((job,), ignore_conditions)
 
     @deprecated(deprecated_in="0.11", removed_in="0.13", current_version=__version__)
     def complete(self, job):
-        """Check if all BaseFlowOperation post-conditions are met.
+        """Check if all BaseFlowOperation postconditions are met.
 
-        :param job:
+        Parameters
+        ----------
+        job : :class:`~signac.contrib.job.Job`
             A :class:`~signac.contrib.job.Job` from the signac workspace.
-        :type job:
-            :class:`~signac.contrib.job.Job`
-        :return:
+
+        Returns
+        -------
+        bool
             Whether the group is complete (all contained operations are
             complete).
-        :rtype:
-            bool
+
         """
         return self._complete((job,))
 
     def add_operation(self, name, operation, directives=None):
-        """Add an operation to the FlowGroup.
+        """Add an operation to the :class:`~.FlowGroup`.
 
-        :param name:
+        Parameters
+        ----------
+        name : str
             The name of the operation.
-        :type name:
-            str
-        :param operation:
-            The workflow operation to add to the FlowGroup.
-        :type operation:
-            :class:`BaseFlowOperation`
-        :param directives:
-            The operation specific directives.
-        :type directives:
-            dict
+        operation : :class:`BaseFlowOperation`
+            The workflow operation to add to the :class:`~.FlowGroup`.
+        directives : dict
+            The operation specific directives. (Default value = None)
+
         """
         self.operations[name] = operation
         if directives is not None:
             self.operation_directives[name] = directives
 
     def isdisjoint(self, group):
-        """Return whether two groups are disjoint (do not share any common operations).
+        """Return whether two groups are disjoint.
+
+        Groups are disjoint if they do not share any common operations.
 
-        :param group:
+        Parameters
+        ----------
+        group : :class:`flow.project.FlowGroup`
```

Comment:
There is a little inconsistency with respect to the reference 
For example, :class:`~.FlowGroup` v/s :class:`~flow.project.FlowGroup`
Do we need to fix it?